### ISSUE TITLE
Flameshot not working with KDE Plasma (Wayland)

### ISSUE THREAD

**Issue by LiterallyJohnny** [flameshot-org/flameshot#1390]
**Flameshot version**

Flameshot v0.8.5
Compiled with Qt 5.15.1

**Describe the bug**

Flameshot does not launch the GUI on KDE Plasma (Wayland), on Arch Linux. `pipewire`, `xdg-desktop-portal`, and `xdg-desktop-portal-kde` are all installed, but it still doesn't launch.

**To Reproduce**

- Boot into KDE Plasma (Wayland)
- Verify `pipewire`, `xdg-desktop-portal`, and `xdg-desktop-portal-kde` are all installed.
- Run `flameshot gui` in a terminal
- Note that the terminal prompt immediately gets ready for the input of another command, but Flameshot still doesn't launch

**Expected behavior**

Flameshot should have opened the screenshot GUI as it would have in Xorg.

**System Information**

- OS: Arch Linux
- Display server: Wayland
- DE: KDE Plasma (Wayland)

**Comment by mmahmoudian** [comment]
We have a new [Release Candidate](https://github.com/flameshot-org/flameshot/releases/tag/v0.9.0.rc2) in which the Wayland support has been drastically improved. I would be thankful if you test the new version and let us know if your issue is resolved. I suggest you download the [AppImage version](https://github.com/flameshot-org/flameshot/releases/download/v0.9.0.rc2/Flameshot-0.9.0.rc2.x86_64.AppImage):

```sh
wget https://github.com/flameshot-org/flameshot/releases/download/v0.9.0.rc2/Flameshot-0.9.0.rc2.x86_64.AppImage
chmod +x Flameshot-0.9.0.rc2.x86_64.AppImage
Flameshot-0.9.0.rc2.x86_64.AppImage gui
```

**Comment by borgmanJeremy** [comment]
this is actually a duplicate of #1380 . The user on that issue has identified a fix I need to test and merge into 0.9 still.

**Comment by LiterallyJohnny** [comment]
Ah, alright then. Guess I'll wait then.
